bee-node/bee-plugin/bee-plugin-version-checker/src/lib.rs

```diff
@@ -5,43 +5,108 @@
 
 #![warn(missing_docs)]
 
+mod release_info;
+
+use release_info::{ReleaseInfo, ReleaseInfoBuilder};
+
 use bee_runtime::{node::Node, shutdown_stream::ShutdownStream, worker::Worker};
 
 use async_trait::async_trait;
 use futures::StreamExt;
-use log::info;
 use tokio::time::interval;
 use tokio_stream::wrappers::IntervalStream;
 
 use std::{convert::Infallible, time::Duration};
 
 const CHECK_INTERVAL_SEC: u64 = 3600;
 
+type VersionCheckerConfig = String;
+
 /// Version checker plugin.
 #[derive(Default)]
 pub struct VersionCheckerPlugin;
 
 #[async_trait]
 impl<N: Node> Worker<N> for VersionCheckerPlugin {
-    type Config = ();
+    type Config = VersionCheckerConfig;
     type Error = Infallible;
 
-    async fn start(node: &mut N, _config: Self::Config) -> Result<Self, Self::Error> {
+    async fn start(node: &mut N, current_version: Self::Config) -> Result<Self, Self::Error> {
         node.spawn::<Self, _, _>(|shutdown| async move {
-            info!("Running.");
+            log::info!("Version checker running.");
```

Comment:
Can you just use `Running` and `Stopped` for consistency ? The name is kind of already in the log metadata.

Reply:
Sure :+1: